sdk/README.md

```diff
@@ -0,0 +1,27 @@
+# SuiNS SDK
+
+> Note 1: You do not need to use the SDK for name resolution (name -> address, address -> name).
+> That is already covered by JSONRPC & GraphQL.
+
+> SuiNS SDK is mostly functional on testnet, and will become available on mainnet after the release
+> of subnames.
+
+SuiNS SDK is a convenient wrapper for querying more detailed information and building transactions
+towards the Name Service.
+
+You can find docs [by clicking here](https://docs.suins.io).
+
+## Testing
```

Comment:
The readme is what gets published to npm, so I might move docs on how to test the sdk into a different file since that won't be relevant to users looking at the package on npm

Reply:
Fair point, indeed!